.github/workflows/haskell.yml

```diff
@@ -645,3 +645,26 @@ jobs:
           echo 'The diff must not contain any `undefined` values'
           false
         fi
+
+  changelog-lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install changelog linter
+      run: |
+        cd "$(mktemp -d)"
+        cat <<EOF >cabal.project
+        source-repository-package
+          type: git
+          location: https://github.com/neilmayhew/changelogs.git
```

Comment:
I've created a placeholder repository where you can put this tool and all other functionality written in the future (in Haskell :wink:) that we can use for releasing, versioning, etc. 
https://github.com/input-output-hk/cardano-ledger-release-tool
It doesn't have to have the final structure or naming, we can organize that part later.
CC @Lucsanszky This is also the repo that you can use to start putting related functionality that you will be working on.

Reply:
I haven't moved the tool to this repo yet, but I'd like to merge this as-is before the reformatting changes become stale. I'll make another PR after I've moved the tool.